spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -14,18 +14,27 @@ def verify_identity_with_doc_auth
     expect(page).to have_current_path('/verify/doc_auth/upload')
 
     click_on 'Upload from your computer'
-    expect(page).to have_current_path('/verify/doc_auth/front_image')
 
-    click_doc_auth_fallback_link
+    if current_path == '/verify/doc_auth/document_capture'
+      # React-based document capture flow is enabled
+      attach_file 'file-input-1', File.expand_path('spec/fixtures/ial2_test_credential.yml')
+      attach_file 'file-input-2', File.expand_path('spec/fixtures/ial2_test_credential.yml')
+      click_on 'Submit'
+    else
```

Comment:
the changes are clearer if whitespace is ignored, it was just a big indentation

https://github.com/18F/identity-idp/pull/4341/files?diff=split&w=1#diff-4d50b19dfdeb945dac423c227f033c882fe6f3513fea506075060a68e5593bcb